registry/storage/driver/s3-aws/s3.go

```diff
@@ -138,6 +140,14 @@ func FromParameters(parameters map[string]interface{}) (*Driver, error) {
 		secretKey = ""
 	}
 
+	backend, ok := parameters["backend"]
+	if !ok {
+		backend = "aws"
+	}
+
+	regionEndpoint, ok := parameters["regionendpoint"]
```

Comment:
@arschles Made the change. Also the travis is now running the tests.the existing tests will test the new code.